include/chemfiles/files/BasicFile.hpp

```diff
@@ -18,12 +18,8 @@ class BasicFile final: public TextFile {
     /// `'r'` or `'a'` mode.
     BasicFile(const std::string& filename, File::Mode mode);
```

Comment:
Maybe rename to "uncompressed"/"flat" text file or the like. I'm not sure what Basic means anymore.

Reply:
Good idea =)

Reply:
I like 'Flat', that's at least my lab uses to describe files that are not compressed and not binary.